api/v1/routes/permissions/roles.py

```diff
@@ -31,29 +34,22 @@ def assign_role(
     return role_service.assign_role_to_user(db, org_id, user_id, request.role_id)
 
 
-@role_perm.delete("/roles/{role_id}", tags=["delete role"], response_model=ResponseModel)
+@role_perm.delete("/roles/{role_id}", tags=["delete role"], response_model=success_response)
 def delete_role(
     role_id: str, 
     db: Session = Depends(get_db),
     current_user: User = Depends(user_service.get_current_user)
 ):
-    result = role_service.delete_role(db, role_id)
+    try:
```

Comment:
No need for the try block. Exception handling is taken care of. Just return the success response